provider/terraform.rb

```diff
@@ -175,16 +169,16 @@ def generate_operation(output_folder, _types, version_name)
       async = @api.objects.map(&:async).compact.first
 
       data = build_object_data(@api.objects.first, output_folder, version_name)
-      dir = data[:version] == 'beta' ? 'google-beta' : 'google'
-      target_folder = File.join(data[:output_folder], dir)
-
-      generate_resource_file(data.clone.merge(
-                               async: async,
-                               object: @api.objects.first,
-                               default_template: 'templates/terraform/operation.go.erb',
-                               out_file: File.join(target_folder,
-                                                   "#{product_name}_operation.go")
-                             ))
+      dir = data.version == 'beta' ? 'google-beta' : 'google'
+      target_folder = File.join(data.output_folder, dir)
+
+      new_data = data.clone
```

Comment:
new_data instead of data_new? They should be consistent regardless, and hopefully with a name more useful than `new`